src/script/interpreter.cpp

```diff
@@ -75,7 +75,8 @@ static void CleanupScriptCode(CScript &scriptCode,
                               uint32_t flags) {
     // Drop the signature in scripts when SIGHASH_FORKID is not used.
```

Comment:
This comment is basically useless. Can see this from the code. What is the reason for it?

Reply:
There's also `isReserved`; I'm being a bit conservative here.

Reply:
Will remove; it's also a bit confusing now.